docs/specs/#1984 - Apps and Packages Spec/spec.md

```diff
@@ -0,0 +1,118 @@
+---
+author: Sharla Soennichsen @shakersMSFT & Jordi Adoumie @joadoumie
+created on: <2024-01-30>
+last updated: <2024-01-30>
+issue id: [#1984](https://github.com/microsoft/devhome/issues/1984) 
```

Comment:
This throws a yaml error?

Reply:
Oh what! weird! I was just following the template. I'll have to revisit this then.

Reply:
fixed :)